src/tpm2/MathOnByteBuffers.c

```diff
@@ -103,19 +103,29 @@ UnsignedCompareB(
 /* -1 if a < b */
 int
 SignedCompareB(
-	       const UINT32     aSize,         // IN: size of a
-	       const BYTE      *a,             // IN: a buffer
-	       const UINT32     bSize,         // IN: size of b
-	       const BYTE      *b              // IN: b buffer
-	       )
+    const UINT32     aSize,         /*  IN: size of a */
+    const BYTE      *a,             /*  IN: a buffer */
+    const UINT32     bSize,         /*  IN: size of b */
+    const BYTE      *b              /*  IN: b buffer */
+    )
```

Comment:
Can you leave this part here untouched so it remains as close to the specs as possible.